ui/candidate-portal/src/app/components/profile/view/tab/tasks/task/candidate-task.component.html

```diff
@@ -33,6 +33,12 @@ <h5 class="text-alert">{{ 'TASKS.TASK.ABANDONEDDATE' | translate }} {{selectedTa
   </div>
 </div>
 
+<ng-container *ngIf="selectedTask.task.name === 'duolingoTest'">
+  <h5>You are receiving an invitation to take the Duolingo English Test for free! Accept the coupon below to get started.</h5>
+  <p>Your certified results will be sent to Talent Beyond Boundaries (TBB) after you complete the test.</p>
+  <a class="btn btn-sm btn-primary" target="_blank" [href]="'https://example.com/' + candidate.id">Accept Offer</a>
```

Comment:
Can someone point me on where can I add those to the translations? I couldn't find the translations files.

Reply:
We don't actually translate task descriptions as they come from the database as fields. However since this is hard coded it could be translated. Translations are updated using translation keys and then we update those via the Settings on the Admin Portal. I can show you if you'd like just give me a slack @MuhannedNoman :)

Reply:
Actually I've had a thought!! Maybe we'd have more flexibility if we made task descriptions rich text, so that we can add html to the flyway and it would display as rich text in the task? That would reduce the need to hard code text for particular tasks, which you needed to do here to add a button, and allow us to insert a link instead? What do you think? I originally wrote this tasks code before we used rich text, but now we use it in our chats and other descriptive fields, so perhaps we could use it here?